pom.xml

```
@@ -63,7 +63,7 @@
   </scm>
 
   <properties>
-    <revision>0.8.8</revision>
+    <revision>0.8.10</revision>
     <changelist>-SNAPSHOT</changelist>
```

should be a large number. 
```suggestion
    <changelist>999999-SNAPSHOT</changelist>
```